source/extensions/filters/http/cache/http_cache_utils.h

```diff
@@ -1,34 +0,0 @@
-#pragma once
```

Comment:
You can do a `git mv` to preserve the change history when renaming files as well as making it possible to see what's changed in the files. It's also helpful for clarity in the review to rename things in a commit separate from any functionality change (same PR, just a separate commit) to make the resulting diff as clean as it can be.

Reply:
Thanks for the tip! In this case I rewrote the whole file, the only thing remaining from http_cache_utils is the `httpTime` function. So I do not think it should matter for this specific case. However, is there a way to revert these changes and separate them into different commits?

Reply:
After a bit of reading I figured `git mv` will preserve the change history and the file rename in case the changes in the file are minor enough that git can keep track of them. I should have had a separate commit to rename the file like you mentioned.